src/Groups/gsets.jl

```diff
@@ -73,6 +73,7 @@ end
 
 """
     acting_group(Omega::GSetByElements)
+    acting_group(Omega::GSetBySubgroupTransversal)
```

Comment:
Shouldn't this be part of the general `GSet` interface and thus documented as such?
```suggestion
    acting_group(Omega::GSet)
```